import_factors_exio/README.md

```diff
@@ -1,5 +1,71 @@
 # Generating Import Factors from Exiobase
-To generate import factors from exiobase, run the script [generate_import_factors.py](generate_import_factors.py) 
+
+## BEA API Access
+
+To pull data on service imports from the US Bureau of Economic Analysis (BEA) by service category, country, and year, register for the BEA API at https://apps.bea.gov/api/signup/.
+
+After doing so, you'll be provided with an API key via email.
+
+Create the file "../import_factors_exio/API/BEA_API_key.yaml" and add your API key.
+
+## Pull Data from Exiobase and BEA
+
+To generate import factors from exiobase using BEA, run the following. Generating 6 years takes about 70 minutes. The resulting 2017-2022 year files reside in the output subfolder, so you can avoid rerunning if those serve your needs. (Total output: 168.9 MB for 37 items)
+
+	python generate_import_factors.py
+
+You can optionally create a virtual environment first and install the following python libraries:
+<!-- Let us know if you also need to add `pip install path` --->
+
+	python3 -m venv env
+	source env/bin/activate
+	pip install pandas
+	pip install pyyaml
+	pip install pymrio
+	pip install pathlib
```

Comment:
pathlib is a core python package so does not need to be installed with pip.

I'm also fine with supplying an env file here instead